etc/features.yaml

```diff
@@ -85,4 +85,6 @@
     inspec-reporter-progress-bar:
       description: Use progress bar streaming reporter
     inspec-reporter-child-status:
-      description: Child status reporter used in inspec parallel reporting.
\ No newline at end of file
+      description: Child status reporter used in inspec parallel reporting.
+    inspec-profile-mandatory-signing:
+      flag: CHEF_PREVIEW_MANDATORY_PROFILE_SIGNING
```

Comment:
Please try to keep your work on separate PRs.

I'd like you to add a feature-neutral test flag. Say, CHEF_PREVIEW_TEXT_FIXTURE_FLAG. That way, you have a feature you can test with on this PR, and it has nothing to do with the other work going on elsewhere.

Then we block the mandatory signing pr until this one merges (should be fast).

Then, on the mandatory signing PR, you should introduce a new feature named for mandatory profile signing. That way all the impact of mandatory signing stays on the other PR.